### ISSUE TITLE
Function  `docker.compose.down`  does not accept optional list of service names

### ISSUE THREAD

**Issue by MisterOwlPT** [gabrieldemarmiesse/python-on-whales#570]
Hi 😃 

The Docker _Compose_ CLI allows the command `down` to be passed a list of service names  (similar to other commands `up` and `rm`).

```bash
$ docker compose down --help

Usage:  docker compose down [OPTIONS] **[SERVICES]**
Stop and remove containers, networks
...
```

I noticed that `python_on_whales` does not currently support this feature, which I personally need for a personal use case.
Calling `docker.compose.down()` simply stops _all_ declared service containers. 

If you agree, I can submit a PR with a logic similar to [this](https://github.com/gabrieldemarmiesse/python-on-whales/blob/master/python_on_whales/components/compose/cli_wrapper.py#L1007) such that if service names are provided, then only the related containers are stopped. Otherwise, if no list of service names is provided, _all_ service containers are stopped.

**Comment by gabrieldemarmiesse** [comment]
Thank you for opening this feature request! When I made the first iteration of python-on-whales, this was not available. Feel free to make a pull request, I'll take a look at it.

**Comment by gabrieldemarmiesse** [comment]
Closing as fixed by #571
